src/components/pages/course-view/CourseViewContainer.jsx

```diff
@@ -1,56 +1,50 @@
 import React, { useState } from 'react';
 import { Card } from 'antd';
+import { ModuleView } from '../index';
 import styled from 'styled-components';
 
+//Component Styles
 const CourseCard = styled(Card)`
   margin-bottom: 4%;
 `;
 
-const courseData = [
-  {
-    courseName: 'Python Basics',
-  },
-  {
-    courseName: 'Problem Solving',
-  },
-  {
-    courseName: 'Time and Space Complexity',
-  },
-];
+const Description = styled.div`
+  margin-bottom: 2%;
+`;
 
 const CourseViewContainer = props => {
```

Comment:
Overall, this component is BANGING. Good comments, great use of boolean logic for `isExpanded`; it's readable, tight, and makes good real-estate of the data coming in. So far I'm very impressed with your code.